### PR TITLE
from 0.1.40 to 0.2.0

### PR DESCRIPTION
**Pull request by rmgogogo**
we introduced MLMD writer and also renamed MLMD configmap. it requires version bump.

After so many months hard working to introduce so many new feature, also worth a minor version bump

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2872)

### CODE REVIEW

File: sdk/python/kfp/__init__.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.1.40'
+__version__ = '0.2.1'
```
Comment (jingzhang36) [inline sdk/python/kfp/__init__.py]: 0.2.0

File: manifests/gcp_marketplace/chart/kubeflow-pipelines/values.yaml [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-  minio: gcr.io/ml-pipeline/google/pipelines/minio:0.1.31
-  mysql: gcr.io/ml-pipeline/google/pipelines/mysql:0.1.31
-  persistenceagent: gcr.io/ml-pipeline/google/pipelines/persistenceagent:0.1.31
-  proxyagent: gcr.io/ml-pipeline/google/pipelines/proxyagent:0.1.31
-  scheduledworkflow: gcr.io/ml-pipeline/google/pipelines/scheduledworkflow:0.1.31
-  viewercrd: gcr.io/ml-pipeline/google/pipelines/viewercrd:0.1.31
-  visualizationserver: gcr.io/ml-pipeline/google/pipelines/visualizationserver:0.1.31
-  metadataenvoy: gcr.io/ml-pipeline/google/pipelines/metadataenvoy:0.1.31
-  metadatawriter: gcr.io/ml-pipeline/google/pipelines/metadatawriter:0.1.40
+  apiserver: gcr.io/ml-pipeline/google/pipelines/apiserver:dummy
```
Comment (numerology) [inline manifests/gcp_marketplace/chart/kubeflow-pipelines/values.yaml]: Actually I'm curious what's the MKP long-term plan for those fields if they are not functional.

File: manifests/kustomize/base/kustomization.yaml [hunk 3]
```diff
@@ -13,16 +13,16 @@ resources:
 
 images:
 - name: gcr.io/ml-pipeline/api-server
-  newTag: 0.1.40
+  newTag: 0.2.0
```
Comment (numerology) [inline manifests/kustomize/base/kustomization.yaml]: Shall we also bump the version under manifests/kustomize/README.md? Also the one in kubeflow/website?

**Pull request merged**